packages/angular-cli/lib/config/schema.json

```diff
@@ -247,6 +247,19 @@
               "default": true
             }
           }
+        },
+        "serve": {
+          "type": "object",
+          "properties": {
+            "port": {
+              "type": "number",
```

Comment:
We need to add `description` for all new properties. We're going to do a full run through existing ones, but new properties should have proper descriptions. This is because we're going to add support for IDEs that support schema, and those descriptions will be shown to users. Also, the config documentation will be generated from these.